source/binding/AssignmentExpressions.cpp

```diff
@@ -303,6 +304,19 @@ Expression& Expression::convertAssignment(const BindContext& context, const Type
             return *result;
         }
 
+        if (expr.kind == ExpressionKind::Streaming) {
+            if (Bitstream::canBeSource(type, expr.as<StreamingConcatenationExpression>(),
+                                       context)) {
+                // Create implicit streaming concatenation conversion
```

Comment:
I change the comment into
```c++
                // Add an implicit bit-stream casting otherwise types are not assignment compatible.
                // The size rule is not identical to explicit bit-stream casting so a different
                // ConversionKind is used.
```